cli/internal/transformer/transformer.go

```diff
@@ -62,6 +69,9 @@ func (t *RecordTransformer) TransformSchema(sc *arrow.Schema) *arrow.Schema {
 		if _, ok := mdMap["cq:extension:primary_key"]; ok && t.removePks {
 			mdMap["cq:extension:primary_key"] = "false"
 		}
+		if field.Name == "_cq_id" && t.cqIDPrimaryKey {
```

Comment:
```suggestion
		if field.Name == schema.CqIDColumn.Name && t.cqIDPrimaryKey {
```

Reply:
We are not supposed to import the plugin-sdk in the CLI anymore, it's only for plugins to import 🙈 

But we can create a constant in this package maybe